openlibrary/plugins/upstream/account.py

```diff
@@ -325,7 +325,11 @@ def POST(self):
         if error:
             return self.render_error(error, i)
 
-        expires = (i.remember and 3600 * 24 * 7) or ""
+        expires = i.remember and 3600 * 24 * 7
+
+        if email in ['mek@archive.org', 'brenton@archive.org', 'brewster@archive.org']:
```

Comment:
```suggestion
        if email in ('mek@archive.org', 'brenton@archive.org', 'brewster@archive.org'):
```
Tuples:
1. Are faster,
2. Take less memory,
3. Signal the reader that this collection will not change at runtime.

Reply:
this code was never meant to be production, marking as resolved.